admin/templates/register.html

```diff
@@ -0,0 +1,73 @@
+{% load render_bundle from webpack_loader %}
```

Comment:
Probably not part of this PR but both this and the login template could inherit from base.html. If you move the navigation you want hidden in base.html into a block and just pass an empty navigation block from the child templates where you want it hidden. This would give you more reuse and less duplication.